Git-Behind-Proxy.md

```diff
@@ -0,0 +1,38 @@
+# Git Behind A Proxy Server
+
+### Use-cases
+
+You might need to modify `git` commands that accesses (to update them and read from) remote repositories if your internet access is through a [proxy server](https://en.wikipedia.org/wiki/Proxy_server).
```

Comment:
* I believe 'accesses' should be changed to 'access'.
* I would do (to update and read from) and take out 'them'.